integrations/Mouseflow/browser.js

```diff
@@ -0,0 +1,52 @@
+import logger from "../../utils/logUtil";
+class mouseflow {
+  constructor(config) {
+    this.siteId = config.siteID; //1549611
+    this.name = "Mouseflow";
+    this._ready = false;
+  }
+
+  init() {
+    window.mouseflowProjectId = this.siteId;
```

Comment:
take another config **mouseflowHtmlDelay**.
https://support.mouseflow.com/support/solutions/articles/44001539246-delay-the-heatmap-snapshot